doc/cli/operator-sdk_run_ansible.md

```diff
@@ -15,6 +15,7 @@ operator-sdk run ansible [flags]
 ### Options
 
 ```
+      --ansible-roles-path string        (valid just for up local) - Ansible path roles ($ANSIBLE_ROLES_PATH). By default it will be /roles in the current directory
       --ansible-verbosity int            Ansible verbosity. Overridden by environment variable. (default 2)
```

Comment:
@fabianvf it is gen automatically. 
Also, I note that the up local doc has not the Ansible and Helm docs. 
We need to change the code, I mean refractory it in order to allow it to be gen correctly.

Reply:
See; https://github.com/operator-framework/operator-sdk/issues/2369